docs/UserGuide.md

```diff
@@ -65,10 +65,22 @@ TaskHub is a **desktop app for managing contacts, optimized for use via a Comman
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+#### GUI Help: 
+Upon clicking help from the GUI as shown below,
+
+![circled_help_gui](images/circledHelpGUI.png)
+
+a message is displayed, with the link to our user guide. 
 
 ![help message](images/helpMessage.png)
 
+#### CLI Help
+Alternatively, quickly enter a `help` command into TaskHub to display to pop-up shown below.
```

Comment:
Is it meant to be "...display _the_ pop-up shown below."?

Reply:
yeah you're right, that's my bad